<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenFileDialog</class>
 <widget class="QDialog" name="OpenFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Open File:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>BitFormat</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="oneBitButton">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>1 Bit</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="fourBitButton">
          <property name="text">
           <string>4 Bit</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="eightBitButton">
          <property name="text">
           <string>8 Bit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="sixteenBitButton">
          <property name="text">
           <string>16 Bit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="twentyfourBitButton">
          <property name="text">
           <string>24 Bit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="paletteButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Palette</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>oneBitButton</tabstop>
  <tabstop>fourBitButton</tabstop>
  <tabstop>eightBitButton</tabstop>
  <tabstop>sixteenBitButton</tabstop>
  <tabstop>twentyfourBitButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OpenFileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OpenFileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oneBitButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>buttonBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fourBitButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>buttonBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eightBitButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>buttonBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sixteenBitButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>buttonBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>twentyfourBitButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>buttonBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eightBitButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>paletteButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
